app/assets/stylesheets/era_custom.scss

```diff
@@ -16,3 +16,42 @@ div.terms-of-use-custom {
   margin-left: 30px;
   margin-right: 30px;
 }
+
+#content .info-pane p > a {
```

Comment:
Again, I'd move this out of `era_custom` and put it in a `homepage.scss` file instead or something similar